arenadata/Dockerfile.ubuntu

```diff
@@ -22,7 +22,11 @@ RUN set -eux; \
 # To run sshd directly, but not using `/etc/init.d/ssh start`
     mkdir /run/sshd; \
 # Alter precedence in favor of IPv4 during resolving
-    echo 'precedence ::ffff:0:0/96  100' >> /etc/gai.conf
+    echo 'precedence ::ffff:0:0/96  100' >> /etc/gai.conf; \
+# Install allure-behave for behave tests
+	curl https://bootstrap.pypa.io/pip/2.7/get-pip.py | python && pip install allure-behave==2.4.0; \
```

Comment:
ubuntu has the pip version we need in the repository. it's probably worth installing it from there.
It may also be worth clearing the pip cache? `pip2 cache purge`

Reply:
Done